### DISCUSSION TITLE
Backup should be able to specify directories that will not be backed up.

### DISCUSSION THREAD

**Discussion by wugren** [buckyos/CYFS#263] [Ideas]
Sometimes, some configuration directories do not need to be backed up, and it is hoped that they can be excluded through parameters.

**Comment by lurenpluto** [comment]
Currently backup service supports uni backup mode, which supports the following three types of data:
1. objects
2. chunks
3. key data

The key data is some default key directory, from your requirements description, it should be these key directories need to be customized, rather than using the default value, so we can consider adding the corresponding mechanism to support
